var/spack/repos/builtin/packages/opa-psm2/package.py

```diff
@@ -25,6 +25,11 @@ class OpaPsm2(MakefilePackage):
 
     depends_on('numactl')
 
+    # patch to prevent opa-psm2 from adding an additional "usr/"
+    #   subdirectory within the installation prefix, which breaks paths for
+    #   dependent packages like libfabric
+    patch('opa-psm2-install-prefix.patch', when='@11.2.68:') 
```

Comment:
@carns would it be appropriate to add a link here to `https://github.com/mercury-hpc/mercury/issues/289` so that users of this package could track the progress of this issue (and would be able to know when to put an upper bound on this)?

Reply:
Hi @scheibelp , I'm not sure what to put in the comment yet because we don't (yet anyway) have a solid recommendation for what version combinations to use with opa-psm2 and libfabric to get the most stability.  Once we figure something out I'll be happy to add it.

In the meantime, I think this PR is still useful as is, because without it you cannot install the libfabric package with the fabrics=psm2 option (using default package versions).  It would help with testing if ourselves and others could more easily sweep through some permutations.

FWIW, there will probably be another PR soon to update libfabric to 1.8.0, likely from @shanedsnyder.  This brings along several libfabric improvements but doesn't by itself resolve psm2 reliability for us either.

Reply:
Thanks for the detailed response!

Reply:
FYI, the libfabric version bump just submitted in PR #11918.